### PR TITLE
Serverless cost estimator

### PR DESCRIPTION
**Pull request by gemma-shay**
Partially closes https://cockroachlabs.atlassian.net/browse/DOC-4836
Related to https://github.com/cockroachdb/docs/pull/13354

Document the serverless usage cost estimator and add a release note. The movr estimation tutorial also uses this tool.

### CODE REVIEW

**Review by emily-horing — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15045)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @emily-horing and @gemma-shay)

___
*[`cockroachcloud/serverless-cluster-management.md` line 52 at r1](https://reviewable.io/reviews/cockroachdb/docs/15045#-NBm1ShMEVZWLC5U0ki8:-NBm1ShNA_HGTgF6B8w2:bltpdwl) ([raw file](https://github.com/cockroachdb/docs/blob/ecc2c9ea089cd52ea424b7a9de3829e930af42aa/cockroachcloud/serverless-cluster-management.md#L52)):*
> ```Markdown
> 1. Select a time frame in which your workload was active.
> 
>     Your used RUs, used storage, and spend from the selected time frame will be shown along with your monthly cost estimate.
> ```

The free resources are not accounted for in the spend amount. You say it isn't included in the estimate for the month below, but we should note both, maybe just up top here.
___
*[`cockroachcloud/serverless-cluster-management.md` line 54 at r1](https://reviewable.io/reviews/cockroachdb/docs/15045#-NBm0RO_-c9t8hIdWk1X:-NBm0ROa2I9KwqEIjMc5:b6qllap) ([raw file](https://github.com/cockroachdb/docs/blob/ecc2c9ea089cd52ea424b7a9de3829e930af42aa/cockroachcloud/serverless-cluster-management.md#L54)):*
> ```Markdown
>     Your used RUs, used storage, and spend from the selected time frame will be shown along with your monthly cost estimate.
> 
> The monthly cost estimate is calculated using simple extrapolation that assumes your workload during the selected time frame is an accurate representative of your workload over the month. If you haven't been running a workload for at least the length of the selected time frame, your results will likely be inaccurate. It's also important to note that this estimate does not account for the [free resources](learn-about-pricing.html#free-vs-paid-usage) granted to every {{ site.data.products.serverless }} cluster, which you would have to use up before spending anything.
> ```

Nit: "is an accurate _representation_" or "is representative"
Nit: "your results will be inaccurate" - remove "likely"
Nit: "before being charged", since we called it "spend" earlier, it's confusing to now say you won't spend anything.

**Review by emily-horing — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15045#-:-NCMArqCAwDmJcR9SZZx:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @emily-horing)

File: cockroachcloud/serverless-cluster-management.md [hunk 1]
```diff
@@ -38,6 +38,19 @@ The **Overview** page displays details about the selected {{ site.data.products.
...
+{{site.data.alerts.callout_info}}
+This feature is not available if your organization is billed through [Credits](billing-management.html#view-credits-balance).
+{{site.data.alerts.end}}
+
+The monthly cost estimate is calculated using simple extrapolation that assumes your workload during the selected time frame is an accurate representation of your workload over the month. If you haven't been running a workload for at least the length of the selected time frame, your results will be inaccurate.
+
+1. In the **Usage this month** section of your cluster's [**Overview** page](cluster-overview-page.html), click **Estimate usage cost**.
+1. Select a time frame in which your workload was active.
+
+    Your used RUs, used storage, and spend from the selected time frame will be shown along with your monthly cost estimate. The spend and cost estimate do not account for the [free resources](learn-about-pricing.html#free-vs-paid-usage) granted to every {{ site.data.products.serverless }} cluster, which you would have to use up before being charged.
```
[the remark this answers is not in the document]
Comment (rmloveland) [reply cockroachcloud/serverless-cluster-management.md:52]: PS maybe for another PR, but the mention of Request Units / RUs in the table on 'Cluster Overview Page > Usage this month' could also link to the above, I think

**Review by rmloveland — APPROVED** [review]
LGTM, had a couple comments about links that are not blockers

**Pull request merged**